core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/textannotation/Constituent.java

```diff
@@ -179,29 +179,7 @@ public boolean doesConstituentCoverAll(Collection<Integer> tokenIds) {
         return true;
     }
 
-    @Override
-    public boolean equals(Object obj) {
-        if (this == obj)
-            return true;
-
-        if (!(obj instanceof Constituent))
-            return false;
-
-        Constituent that = (Constituent) obj;
-
-        if (this.getStartSpan() < 0) {
-            return false;
-        }
-
-        if (this.attributes == null && that.attributes != null)
-            return false;
-        if (this.attributes != null && that.attributes == null)
-            return false;
-
-        if (this.attributes != null && that.attributes != null)
-            if (!this.attributes.equals(that.attributes))
-                return false;
-
+    public boolean equalsWithoutAttributeEqualityCheck(Constituent that) {
```

Comment:
Can you add some documentation for this? Mentioning the expected use case for using this method.